server/support_packet.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2022-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package main
+
+import (
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v3"
+
+	"github.com/mattermost/mattermost-plugin-calls/server/public"
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/public/plugin"
+)
+
+type SupportPacket struct {
+	Version string `yaml:"version"`
+
+	*public.CallsStats
+}
+
+func (p *Plugin) GenerateSupportData(_ *plugin.Context) ([]*model.FileData, error) {
+	stats, err := p.store.GetCallsStats()
+	if err != nil {
+		return nil, errors.Wrap(err, "Failed to get calls stats")
```

Comment:
Errors should be lowercase.
```suggestion
		return nil, errors.Wrap(err, "failed to get calls stats")
```